blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -221,6 +215,16 @@ public void trackResponse(HttpRequest request, HttpResponse response) {
 
     }
 
+    public void trackDelayedMetricsTenant(String tenantid, List<Metric> delayedMetrics) {
+        if (isTrackingDelayedMetrics) {
+            String logMessage = String.format("[TRACKER][DELAYED METRIC] Tenant sending delayed metrics %s\n",tenantid);
+            for (Metric metric : delayedMetrics) {
+                logMessage += String.format("%s has collectionTime %d\n", metric.getLocator().toString(), metric.getCollectionTime());
```

Comment:
I can convert collectionTime to human readable.

For the amount of delay, that'll takes extra logic because 1) we have to pass in the ingestion time (or add a new timestamp property to each metric, which I was considering), and 2) we have to pull the DELAYED_METRICS_MILLIS from property and calculate the difference for every delayed metrics.  I wasn't sure if I should add that kind of complexity and extra calculation just for this.